### PR TITLE
Added eos_alias type

### PR DESCRIPTION
**Pull request by mmailand**
Added eos_alias type

### CODE REVIEW

**Comment by jerearista** [comment]
Jenkins, test this please

**Comment by mmailand** [comment]
rebased the PR

File: spec/unit/puppet/provider/eos_alias/default_spec.rb [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+                                             name: @name,
+                                             command: @command)
+        provider.create
+        provider.command = @command
+        provider.flush
+        expect(provider.command).to eq(@command)
+      end
+    end
+
+    describe '#command=(value)' do
```
Comment (devrobo) [inline spec/unit/puppet/provider/eos_alias/default_spec.rb]: Can you add a test case for multi-line aliases?

**Comment by mmailand** [comment]
added multiline support in the test and documentation

**Comment by jerearista** [comment]
Jenkins, test this please

**Pull request merged**